pkgs/development/python-modules/lizard/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "lizard";
+  version = "1.17.10";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p2s0wx1l75dsljiyvars7xlljzs1mipm8jaby5y4jvjck6qmmv2";
+  };
+
+  pythonImportsCheck = [
+    "lizard"
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description =
+      "A code analyzer without caring the C/C++ header files. It works with Java, C/C++, JavaScript, Python, Ruby, Swift, Objective C. Metrics includes cyclomatic complexity number etc";
+    homepage = "http://www.lizard.ws";
```

The license needs to be set.

```suggestion
    homepage = "http://www.lizard.ws";
    license = with licenses; [ mit ];
```

---

good catch! I'll add it